src/styles/card.scss

```diff
@@ -0,0 +1,94 @@
+.card {
+  font-family: Roboto, sans-serif;
+  box-sizing: border-box;
+  width: 200px;
+  padding: 16px;
+
+  &__product-image {
+    padding: 16px 3px 0;
+    display: flex;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__product-name {
+    margin: 0;
+    padding-top: 40px;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    color: #060b35;
+  }
+
+  &__product-code {
+    margin: 0;
+    padding-top: 4px;
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+    color: #616070;
+  }
+
+  &__product-rating {
+    padding-top: 16px;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+
+    &-review {
+      margin: 0;
+      display: flex;
+      font-weight: 400;
+      font-size: 10px;
+      line-height: 14px;
+      color: #060b35;
+      align-self: flex-end;
+    }
+  }
+
+  &__product-price {
+    padding-top: 24px;
+    padding-bottom: 16px;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    line-height: 18px;
+
+    &-text {
+      margin: 0;
+      font-weight: 400;
+      font-size: 12px;
+      color: #616070;
+    }
+
+    &-numbers {
+      margin: 0;
+      font-weight: 700;
+      font-size: 16px;
+    }
+  }
+
+  &__product-buy-link {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    border-radius: 5px;
+    width: 100%;
+    height: 40px;
+    background-color: #00acdc;
+
+    text-decoration: none;
+    font-weight: 700;
+    font-size: 14px;
+    line-height: 16px;
+    color: #fff;
+
+    &:hover {
+      background-color: #fff;
+      color: #00acdc;
+      border-style: solid;
+      border-width: 1px;
+      border-color: #00acdc;
+    }
+  }
```

Comment:
The `&:hover` selector on line 86 changes the background color and the text color of the link, which is fine. However, it also adds a border to the link, which can cause the layout to shift because the border takes up space. To avoid this, you should add a transparent border to the link in its normal state. This way, the border is already there, but it's invisible until the link is hovered over.